besu/terraform/aws/ibftScale/files/orion/setup.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+echo $@ > /tmp/args.txt
+
+ORION_VERSION=$1
+ORION_DOWNLOAD_URL=$2
+ORION_BOOT_NODE_IP=$3
+ORION_NODE_IP=$4
+ORION_CLIENT_IP=$5
+# We arent using private IPs because the vpc arent peered. The public IPs act like real world use cases for us anyway
+#ORION_CLIENT_IP=`curl http://169.254.169.254/latest/meta-data/local-ipv4`
```

Comment:
Should this have been left in? Also prefer `$()` style quoting over backticks in bash/shell - it nests more nicely with other quoting types.